corehq/apps/users/models.py

```diff
@@ -382,6 +382,7 @@ class DomainMembership(Membership):
     override_global_tz = BooleanProperty(default=False)
     role_id = StringProperty()
     location_id = StringProperty()
+    assigned_location_ids = StringListProperty()
```

Comment:
Hey Sravan - I think we should design for web users having multiple locations (but I don't think the UI allows for it yet).  What is the lift to make that change for web users?